libs/blocks/merch-cards/merch-cards.js

```diff
@@ -65,25 +66,24 @@ export function parsePreferences(elements) {
   });
 }
 
-async function initMerchCards(config, type, filtered, el, preferences) {
-  let cardsData;
-  let err;
+const localizedPath = (path, config) => `${config?.locale?.prefix ?? ''}${path}`;
 
+const fetchOverrideCard = (path, config) => new Promise((resolve, reject) => {
   try {
-    const res = await fetch(`${config?.locale?.prefix ?? ''}${config.queryIndexCardPath}.json?sheet=${type}`);
-    if (res.ok) {
-      cardsData = await res.json();
-    } else {
-      err = res.statusText || res.status;
-    }
+    fetch(`${localizedPath(path, config)}.plain.html`).then((res) => {
+      if (res.ok) {
+        res.text().then((cardContent) => {
+          resolve({ path, cardContent: /^<div>(.*)<\/div>$/.exec(cardContent.replaceAll('\n', ''))[1] });
```

Comment:
we should make sure that `/^<div>(.*)<\/div>$/.exec(cardContent.replaceAll('\n', ''))[1]` never breaks the base flow.

Reply:
we _are_ in a try catch, see l80